login form modified

Mxyank opened the pull request.

SaraswatGit commented:
@Mxyank  Attach a screenshot/screen recording

Mxyank commented:
![PR merge](https://user-images.githubusercontent.com/69813711/156318016-758a91fc-5931-4e9d-8905-b37d4b90c4de.png)

Mxyank commented:
Yeah,Actually i have changed some line of CSS

On Wed, Mar 2, 2022 at 1:24 PM Tess-Vanta ***@***.***> wrote:

> @Mxyank <https://github.com/Mxyank> can you give the link to the issue
> and also send a screenshot of the change made in the project
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SaraswatGit/PlanZap/pull/128#issuecomment-1056496384>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQUULT5MPRDJCDLSQZV6QKTU54NCDANCNFSM5PWOMBRA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Tess-Vanta commented:
> Yeah,Actually i have changed some line of CSS
> […](#)
> On Wed, Mar 2, 2022 at 1:24 PM Tess-Vanta ***@***.***> wrote: @Mxyank <https://github.com/Mxyank> can you give the link to the issue and also send a screenshot of the change made in the project — Reply to this email directly, view it on GitHub <[#128 (comment)](https://github.com/SaraswatGit/PlanZap/pull/128#issuecomment-1056496384)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AQUULT5MPRDJCDLSQZV6QKTU54NCDANCNFSM5PWOMBRA> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. You are receiving this because you were mentioned.Message ID: ***@***.***>

Were you assigned an issue ?

Mxyank commented:
As i am new to open source so I have not fixed any issue i have just
changes the css alignments in button to look better than previous one

On Wed, 2 Mar 2022, 13:34 Tess-Vanta, ***@***.***> wrote:

> Yeah,Actually i have changed some line of CSS
> … <#m_-4869948364629136114_>
> On Wed, Mar 2, 2022 at 1:24 PM Tess-Vanta *@*.*> wrote: @Mxyank
> <https://github.com/Mxyank> https://github.com/Mxyank
> <https://github.com/Mxyank> can you give the link to the issue and also
> send a screenshot of the change made in the project — Reply to this email
> directly, view it on GitHub <#128 (comment)
> <https://github.com/SaraswatGit/PlanZap/pull/128#issuecomment-1056496384>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/AQUULT5MPRDJCDLSQZV6QKTU54NCDANCNFSM5PWOMBRA
> <https://github.com/notifications/unsubscribe-auth/AQUULT5MPRDJCDLSQZV6QKTU54NCDANCNFSM5PWOMBRA>
> . Triage notifications on the go with GitHub Mobile for iOS
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
> You are receiving this because you were mentioned.Message ID: @.*>
>
> Were you assigned an issue ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SaraswatGit/PlanZap/pull/128#issuecomment-1056514974>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQUULT46OMOC5Z2C7NBHTSLU54OJTANCNFSM5PWOMBRA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Mxyank commented:
I have changed this

On Wed, Mar 2, 2022 at 1:37 PM Mayank agrawal ***@***.***>
wrote:

> As i am new to open source so I have not fixed any issue i have just
> changes the css alignments in button to look better than previous one
>
> On Wed, 2 Mar 2022, 13:34 Tess-Vanta, ***@***.***> wrote:
>
>> Yeah,Actually i have changed some line of CSS
>> … <#m_6669155440766718788_m_-4869948364629136114_>
>> On Wed, Mar 2, 2022 at 1:24 PM Tess-Vanta *@*.*> wrote: @Mxyank
>> <https://github.com/Mxyank> https://github.com/Mxyank
>> <https://github.com/Mxyank> can you give the link to the issue and also
>> send a screenshot of the change made in the project — Reply to this email
>> directly, view it on GitHub <#128 (comment)
>> <https://github.com/SaraswatGit/PlanZap/pull/128#issuecomment-1056496384>>,
>> or unsubscribe
>> https://github.com/notifications/unsubscribe-auth/AQUULT5MPRDJCDLSQZV6QKTU54NCDANCNFSM5PWOMBRA
>> <https://github.com/notifications/unsubscribe-auth/AQUULT5MPRDJCDLSQZV6QKTU54NCDANCNFSM5PWOMBRA>
>> . Triage notifications on the go with GitHub Mobile for iOS
>> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>> You are receiving this because you were mentioned.Message ID: @.*>
>>
>> Were you assigned an issue ?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/SaraswatGit/PlanZap/pull/128#issuecomment-1056514974>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQUULT46OMOC5Z2C7NBHTSLU54OJTANCNFSM5PWOMBRA>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
